problem_2/spec/even_fibonacci_numbers_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'even_fibonacci_numbers'
+
+describe FibonacciCalculator do
+	let (:calculator) { FibonacciCalculator.new(limit) }
+		
+	describe '.add_fib_sequence' do
+			
+		context 'when limit is 100' do
```

Comment:
`context` is usually reserved for tests that require some more explanation. I would `it` instead here